src/OverviewView.js

```diff
@@ -0,0 +1,141 @@
+import { OrthographicView } from '@deck.gl/core';
+import VivView from './VivView';
+import { OverviewLayer } from './layers';
+
+/**
+ * This class generates a OverviewLayer and a view for use in the VivViewer as an overview to a Detailview (they must be used in conjection)
+ * @param {Object} viewState The viewState object.
+ * @param {Object} loader The loader, used for inferring zoom level.
+ * @param {number} detailHeight The height of the detail view.
+ * @param {number} detailWidth The width of the detail view.
+ * @param {number} overviewScale The scale of this viewport relative to the detail.
+ * @param {number} margin The margin to be offset from the the corner of the other viewport.
+ * @param {string} overviewLocation The location of the viewport - one of "bottom-right", "top-right", "top-left", "bottom-left."
+ * */
+export default class OverviewView extends VivView {
+  constructor({
+    viewState,
+    loader,
+    detailHeight,
+    detailWidth,
+    overviewScale,
+    margin = 25,
+    overviewLocation = 'bottom-right'
+  }) {
+    super({ viewState });
+    this.margin = margin;
+    this.loader = loader;
+    this.overviewLocation = overviewLocation;
+    this.detailHeight = detailHeight;
+    this.detailWidth = detailWidth;
+    this._setHeightWidthScale({ detailWidth, overviewScale });
+    this._setXY();
+  }
+
+  _setHeightWidthScale({ detailWidth, overviewScale }) {
+    const { loader } = this;
+    const { numLevels } = loader;
+    const rasterSize = loader.getRasterSize({
```

Comment:
Perhaps the loader method isn't the greatest name, but I'd read this easier as .. 
```javascript
const { width: rasterWidth, height: rasterHeight } = loader.getRasterSize({
```

Reply:
Ok.